portal/templates/redesign/about_new.html

```diff
@@ -0,0 +1,138 @@
+{% extends 'redesign/base_new.html' %}
+{% load staticfiles %}
+{% load app_tags %}
+
+{% block css %}
+{{ block.super }}
+{% endblock css %}
+
+{% block scripts %}
+{{ block.super }}
+{% endblock scripts %}
+
+{% block pageID %}id='homeWrapper'{% endblock %}
+
+{% block content %}
+{{ block.super }}
+
+<div class="banner banner--teacher row">
+    <div class="col-sm-5 col-sm-offset-2">
+        <h1 class="banner__text--primary">About Code for Life</h1>
+        <h4>Code For Life gives everyone the ability to shape technology&rsquo;s future</h4>
+    </div>
+    <div class="col-sm-5">
+        <div class="hexagon"><div class="hexagon-in1"><div class="hexagon-in2 hexagon-in2--about"></div></div></div>
+    </div>
+</div>
+
+<div class="background">
+    <section>
+        <div class="col-sm-6 col-center">
+            <h4>Code For Life is a non profit initiative that delivers free, open-source games that help all students learn computing.</h4>
+        </div>
+        <div class="col-sm-8 col-center">
+            <div class="row row--regular homepage-info">
+                <div class="col-sm-4">
+                    <h1>13%</h1>
+                    <p class="body-text col-sm-11 col-center">of university computer science graduates in the UK are women*. We want both boys and girls to feel inspired and
+                        confident about technology from a young age.</p>
+                </div>
+                <div class="col-sm-4">
+                    <h1>70,000</h1>
+                    <p class="body-text col-sm-11 col-center">Registered users across the world, and growing everyday.</p>
+                </div>
+                <div class="col-sm-4">
+                    <h1>2014</h1>
+                    <p class="body-text col-sm-11 col-center">The year that computing was added to the UK curriculum. We&rsquo;ve been supporting teachers and students ever since.</p>
+                </div>
+            </div>
+        </div>
+        <br>
+    </section>
+    <div class="col-sm-8 col-center">
+        <div class="row row--regular">
+            <div class="col-sm-6">
+                <h3>Ocado Technology</h3><br>
+                <p class="body-text"><a target="_blank" href="http://www.ocadotechnology.com/">Ocado Technology</a> develops the innovative software and hardware that
```

Comment:
You shouldn't need a class called body-text.  Everything should be body-text unless otherwise specified.